dashboard/app/views/pd/application/teacher1819_application_mailer/principal_approval_received.html.haml

```diff
@@ -0,0 +1,24 @@
+%p
+  Hello
+  = @application.first_name + ','
+
+%p
```

Comment:
@tanyaparker ?

Reply:
Yes, I think it's still reasonable to send this mail even if the principal rejects the teacher. That way the teacher knows they don't have to continue following up with the principal, especially because teachers do receive messaging to help follow up with principals if needed.

If a principal rejected the teacher it may be because of other reasons like they're not sure if it would be that specific teacher but are still interested in the program. It would just indicate that there would need to be further investigation and discussion with the regional partner and school.